libs/remix-analyzer/src/solidity-analyzer/modules/similarVariableNames.ts

```diff
@@ -43,13 +44,25 @@ export default class similarVariableNames implements AnalyzerModule {
         if (multipleContractsWithSameName) {
           multipleContractsWithSameNameComments = 'Note: Import aliases are currently not supported by this static analysis.'
         }
-
         const vars: string[] = this.getFunctionVariables(contract, func).map(getDeclaredVariableName)
         this.findSimilarVarNames(vars).map((sim) => {
-          warnings.push({
-            warning: `${funcName} : Variables have very similar names "${sim.var1}" and "${sim.var2}". ${hasModifiersComments} ${multipleContractsWithSameNameComments}`,
-            location: func.node['src']
-          })
+          // check if function is implemented
+          if(func.node.implemented) {
```

Comment:
I'm not sure if we have a linter setup here but you need a space after if.

Reply:
Yeah. We have the issue opened https://github.com/ethereum/remix-project/issues/559 . If you are fine, can we do this with that issue implementation?